server/chat-commands/admin.ts

```diff
@@ -602,16 +608,57 @@ export const commands: ChatCommands = {
 			this.checkCan('lockdown');
 		}
 
-		let buf = `<strong>${process.pid}</strong> - Main<br />`;
+		const processes = new Map<string, ProcessData>();
+
+		const psOutput = child_process.execSync('ps -o pid,%mem,time,command', {cwd: `${__dirname}/../..`}).toString();
```

Comment:
Is `cwd` necessary? `ps` isn't touching the filesystem, is it?

Reply:
I'm not quite sure? I figured better safe than sorry, especially since `bash` does it above as well.

Reply:
Also, why not `%cpu`? That's the whole reason we wanted it in he first place, right?

Reply:
lol yeah i switched them - fixed.